config_example.yaml

```diff
@@ -6,6 +6,7 @@ customfields: # custom fields are added to falco events
   Bkey: "BValue"
   Ckey: "CValue"
 checkCert: true # check if ssl certificate of the output is valid (default: true)
+mutualtlsfilespath: "/mutualtls-files/" # folder which will used to store client.crt, client.key and ca.crt files for mutual tls
```

Comment:
I prefer to let people decide the names for their certs and keys. 3 parameters should be there so.

Reply:
There will be only one client cert, one ca cert and one client key. Does it really matter the name of the files mounted on the container? What is the point?

Reply:
I just don't like to enforce people to name external resources for falcosidekick, IMHO we should be able to allow them to choose what they want as long they have a key, a cert and a ca.

Reply:
Updated README to show how to use this feature using specified filenames convention